torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -2060,6 +2084,42 @@ Tensor* TensorExprKernel::computeConv2d(const torch::jit::Value* v) {
   return new Tensor(ResultBuf.node(), s);
 }
 
+Tensor* TensorExprKernel::computeMatmul(const torch::jit::Value* v) {
+  const Node* n = v->node();
+  auto const& shape = sizesForValue(v);
+  Dtype dtype = kFloat;
+  auto maybe_stype = findDtypeForValue(v);
```

Comment:
What is maybe_stype?

Reply:
"maybe" is for optionality of returned value, "stype" is for scalar-type. I think we've been using this name in surrounding code too.